src/main.cpp

```diff
@@ -86,6 +83,9 @@ int main(int argc, const char** argv)
         sopts.http_connection_timeout = server_conf.get<double>("http-connection-timeout");
         sopts.workers_count = server_conf.get<int>("workers-count");
         sopts.worker_queue_len = server_conf.get<int>("worker-queue-len");
+        const boost::property_tree::ptree& cryptonode_conf = config.get_child("cryptonode");
```

Comment:
Have you modified default config.ini file?

Reply:
No, I have not. These three lines that deal with "cryptonode" existed before me, I have just  moved them. If you think I should, please let me know.

Reply:
Ok